test/ApplicationSpec.scala

```diff
@@ -0,0 +1,30 @@
+import org.specs2.mutable._
```

Comment:
Please don't use wildcard imports, it makes it harder for other people to figure out what the code is doing.